tensorboard/webapp/metrics/effects/metrics_effects_test.ts

```diff
@@ -778,4 +882,87 @@ describe('metrics effects', () => {
       }
     });
   });
+
+  describe('utilities', () => {
+    describe('generateMultiRunTagsToEidMapping', () => {
+      it('does not map image plugin data', () => {
+        const runToEid = {
+          run1: 'eid1',
+          run2: 'eid2',
+          run3: 'eid1',
+        };
+        const tagMetadata = {
+          images: {
+            tagDescriptions: {},
+            tagRunSampledInfo: {
+              tagA: {
+                run1: {maxSamplesPerStep: 1},
+                run2: {maxSamplesPerStep: 1},
+              },
+              tagB: {
+                run3: {maxSamplesPerStep: 1},
+              },
+            },
+          },
+        };
+        expect(
+          TEST_ONLY.generateMultiRunTagsToEidMapping(
+            tagMetadata as any,
+            runToEid
+          )
+        ).toEqual({});
+      });
+
+      it('does not map histogram data', () => {
+        const runToEid = {
+          run1: 'eid1',
+          run2: 'eid1',
+          run3: 'eid2',
+        };
+        const tagMetadata = {
+          histograms: {
+            tagDescriptions: {},
+            tagToRuns: {
+              tagA: ['run1'],
+              tagB: ['run2', 'run3'],
+            },
+          },
+        };
+
+        expect(
+          TEST_ONLY.generateMultiRunTagsToEidMapping(
+            tagMetadata as any,
+            runToEid
+          )
+        ).toEqual({});
+      });
+
+      it('maps scalar data', () => {
```

Comment:
Would it be worth adding one additional test that includes tagMetadata for all of scalars, histograms, and images?

Reply:
I think it's a little redundant but I've added one to be safe.